### ISSUE TITLE
Exception when typing 'Array.ofSeq' in F# program

### ISSUE THREAD

**Issue by bjornbaverfjord** [dotnet/roslyn#26492]
**Version Used**: 
Microsoft Visual Studio Community 2017 
Version 15.6.7
VisualStudio.15.Release/15.6.7+27428.2043
Microsoft .NET Framework
Version 4.7.02556

**Steps to Reproduce**:
1. Load project or create a new equivalent F# project.
2. Type `Array.ofSeq` on a new line at the end of Program.fs
The timing is critical, it has to be typed very fast but not too fast.
`Array.` can be enough, or ` A<TAB>.<TAB>` if the auto complete has been used before so it selects the right items.

**Expected Behavior**:
I expect the text I type to appear without errors.

**Actual Behavior**:
Exception:
```
System.ArgumentOutOfRangeException: 
Specified argument was out of the range of valid values. 
Parameter name: span
 at Microsoft.VisualStudio.Text.SnapshotSpan..ctor(ITextSnapshot snapshot, Span span)
 at Microsoft.CodeAnalysis.Editor.Implementation.Classification.SyntacticClassificationTaggerProvider.TagComputer.AddClassifiedSpansForPreviousTree[TClassificationService](IClassificationDelegationService`1 delegationService, TClassificationService classificationService, SnapshotSpan span, ITextSnapshot lastSnapshot, Document lastDocument, List`1 classifiedSpans)
 at Microsoft.CodeAnalysis.Editor.Implementation.Classification.SyntacticClassificationTaggerProvider.TagComputer.AddClassifiedSpans[TClassificationService](IClassificationDelegationService`1 delegationService, TClassificationService classificationService, SnapshotSpan span, List`1 classifiedSpans)
 at Microsoft.CodeAnalysis.Editor.Implementation.Classification.SyntacticClassificationTaggerProvider.TagComputer.GetTags[TClassificationService](NormalizedSnapshotSpanCollection spans, HostLanguageServices languageServices, IClassificationDelegationService`1 delegationService)
 at Microsoft.CodeAnalysis.Editor.Implementation.Classification.SyntacticClassificationTaggerProvider.TagComputer.GetTags(NormalizedSnapshotSpanCollection spans)
 at Microsoft.CodeAnalysis.Editor.Implementation.Classification.SyntacticClassificationTaggerProvider.Tagger.GetTags(NormalizedSnapshotSpanCollection spans) at Microsoft.VisualStudio.Text.Tagging.Implementation.TagAggregator`1.<GetTagsForBuffer>d__39.MoveNext()
 --- End of stack trace from previous location where exception was thrown ---
 at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
````

[project.zip](https://github.com/dotnet/roslyn/files/1959094/project.zip)

**Comment by gafter** [comment]
F# is not part of Roslyn. Please file your issue at https://github.com/fsharp/fsharp

**Comment by cartermp** [comment]
@gafter The stack trace only has Roslyn editor code present as shown in the error message, which is why I suggested that @bjornbaverfjord post here rather than in VisualFSharp. Can this be re-opened?

**Comment by sharwell** [comment]
Duplicate of #24410 (fixed for 15.7)

**Comment by cartermp** [comment]
Thanks - I think @bjornbaverfjord mentioned that he couldn't see this in 15.7 so that probably confirms this is a dupe
